bogota-abril-2019/css-magazine-cover/jose-dominguez/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>TopGear Magazine</title>
+    <link href="web-gallery/style.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Anton|Open+Sans+Condensed:300" rel="stylesheet">
+</head>
+
+<body>
+    <div id="main_wrapper">
+        <div id="header_container">
+            <header>
+                &quot;The lobster. Why would you deliberately make a creature that's ugly, stupid and delicious?&quot;
+                <span class="white">Clarkson on Creation</span>
+            </header>
+            <img id="person" src="web-gallery/jeremy-clarkson.png">
+        </div>
+
+        <div id="logo_container"></div>
+
+        <div id="text_container">
+            <span id="title_teaser">
+                IT'S THE BATTLE OF THE
+            </span>
+
+            <div id="big_teaser">
+                SPIDERS
+            </div>
+            <div id="quote">
+                <div>THE FERRARI PICKS THE LOCK</div>
+                <div>SLIPS BETWEEN THE PHISICS. THE</div>
+                <div>McLAREN JUST KICKS THE DOORS IN</div>
```

Comment:
1. Uppercase unnecessarym please check https://css-tricks.com/almanac/properties/t/text-transform/
2. Semantic inconsistency. all text should be in the same <p> since is same text.
Example:

`<div>THE FERRARI PICKS THE LOCK<br> SLIPS BETWEEN THE PHISICS. THE <br> McLAREN JUST KICKS THE DOORS IN</div>`

Reply:
Done